schema/classes/Asset.yaml

```diff
@@ -0,0 +1,61 @@
+"@context":
+  schema: http://schema.org/
+  rdf: http://www.w3.org/1999/02/22-rdf-syntax-ns#
+  rdfs: http://www.w3.org/2000/01/rdf-schema#
+  xsd: http://www.w3.org/2001/XMLSchema#
+  owl: http://www.w3.org/2002/07/owl#
+  void: http://rdfs.org/ns/void#
+  dcterms: http://purl.org/dc/terms/
+  dcat: http://www.w3.org/ns/dcat#
+  dcmitype: http://purl.org/dc/dcmitype/
+  dandi: http://schema.dandiarchive.org/
+  prov: http://www.w3.org/ns/prov#
+  crypto: http://id.loc.gov/vocabulary/preservation/cryptographicHashFunctions/
+  md5: crypto:md5
+  sha1: crypto:sha1
+  sha256: crypto:sha256
+  sha512: crypto:sha512
+  attributedTo: prov:wasAttributedTo
+"@graph":
+  - "@id": dandi:Asset
+    "@type": rdfs:Class
+    rdfs:comment: An item describing the contents of a binary blob or file.
+    rdfs:label: Asset
+  - "@id": schema:name
+    schema:domainIncludes: dandi:Asset
+  - "@id": schema:contentUrl
+    schema:domainIncludes: dandi:Asset
+  - "@id": schema:contentSize
+    schema:domainIncludes: dandi:Asset
+  - "@id": schema:encodingFormat
+    schema:domainIncludes: dandi:Asset
+  - "@id": md5
+    schema:domainIncludes: dandi:Asset
+  - "@id": sha1
+    schema:domainIncludes: dandi:Asset
+  - "@id": sha256
+    schema:domainIncludes: dandi:Asset
+  - "@id": sha512
+    schema:domainIncludes: dandi:Asset
+  - "@id": schema:schemaVersion
```

Comment:
what schema does the schemaVersion refer to? the schema for what?

Reply:
we can decide whether this is essential for now for all entities. but in general it may be most useful at the dandiset level.  this would refer to the dandi schema version. think of it as tagging this spec as 1.0.0